README.rst

```diff
@@ -45,6 +50,26 @@ For a manual install:
 * run ``python manage.py migrate djangocms_picture``
 
 
+Configuration
+-------------
+
+Note that the provided templates are very minimal by design. You are encouraged
+to adapt and override them to your project's requirements.
+
+You can override the alignment styles through the following setting:
+
+    DJANGOCMS_PICTURE_ALIGN = [
```

Comment:
@czpython I left the app name as ``djangocms_picture`` and just renamed the text references to ``image``. Renaming causes a lot of problem that I do not want to address yet.